### ISSUE TITLE
Mistake in Go Installation Wiki

### ISSUE THREAD

**Issue by xiruizhao** [librespeed/speedtest#323]
@maddie 
In https://github.com/librespeed/speedtest/wiki/Installation-(Go)#clone-the-repository
`git checkout -b go` creates a new local branch. It should be `git checkout go`.
In addition, is there a way to make `go get github.com/librespeed/speedtest` work?

**Comment by xiruizhao** [comment]
```
../go/src/github.com/librespeed/speedtest/web/web.go:13:2: cannot find package "github.com/go-chi/chi" in any of:
	/usr/lib/go-1.10/src/github.com/go-chi/chi (from $GOROOT)
	/home/ubuntu/go/src/github.com/go-chi/chi (from $GOPATH)
../go/src/github.com/librespeed/speedtest/web/web.go:14:2: cannot find package "github.com/go-chi/chi/middleware" in any of:
	/usr/lib/go-1.10/src/github.com/go-chi/chi/middleware (from $GOROOT)
	/home/ubuntu/go/src/github.com/go-chi/chi/middleware (from $GOPATH)
../go/src/github.com/librespeed/speedtest/web/web.go:15:2: cannot find package "github.com/go-chi/cors" in any of:
	/usr/lib/go-1.10/src/github.com/go-chi/cors (from $GOROOT)
	/home/ubuntu/go/src/github.com/go-chi/cors (from $GOPATH)
../go/src/github.com/librespeed/speedtest/web/web.go:16:2: cannot find package "github.com/go-chi/render" in any of:
	/usr/lib/go-1.10/src/github.com/go-chi/render (from $GOROOT)
	/home/ubuntu/go/src/github.com/go-chi/render (from $GOPATH)
../go/src/github.com/librespeed/speedtest/database/bolt/bolt.go:12:2: cannot find package "github.com/go-sql-driver/mysql" in any of:
	/usr/lib/go-1.10/src/github.com/go-sql-driver/mysql (from $GOROOT)
	/home/ubuntu/go/src/github.com/go-sql-driver/mysql (from $GOPATH)
../go/src/github.com/librespeed/speedtest/results/telemetry.go:21:2: cannot find package "github.com/golang/freetype" in any of:
	/usr/lib/go-1.10/src/github.com/golang/freetype (from $GOROOT)
	/home/ubuntu/go/src/github.com/golang/freetype (from $GOPATH)
../go/src/github.com/librespeed/speedtest/results/telemetry.go:22:2: cannot find package "github.com/golang/freetype/truetype" in any of:
	/usr/lib/go-1.10/src/github.com/golang/freetype/truetype (from $GOROOT)
	/home/ubuntu/go/src/github.com/golang/freetype/truetype (from $GOPATH)
../go/src/github.com/librespeed/speedtest/database/postgresql/postgresql.go:9:2: cannot find package "github.com/lib/pq" in any of:
	/usr/lib/go-1.10/src/github.com/lib/pq (from $GOROOT)
	/home/ubuntu/go/src/github.com/lib/pq (from $GOPATH)
../go/src/github.com/librespeed/speedtest/results/telemetry.go:23:2: cannot find package "github.com/oklog/ulid/v2" in any of:
	/usr/lib/go-1.10/src/github.com/oklog/ulid/v2 (from $GOROOT)
	/home/ubuntu/go/src/github.com/oklog/ulid/v2 (from $GOPATH)
../go/src/github.com/librespeed/speedtest/config/config.go:4:2: cannot find package "github.com/sirupsen/logrus" in any of:
	/usr/lib/go-1.10/src/github.com/sirupsen/logrus (from $GOROOT)
	/home/ubuntu/go/src/github.com/sirupsen/logrus (from $GOPATH)
../go/src/github.com/librespeed/speedtest/config/config.go:5:2: cannot find package "github.com/spf13/viper" in any of:
	/usr/lib/go-1.10/src/github.com/spf13/viper (from $GOROOT)
	/home/ubuntu/go/src/github.com/spf13/viper (from $GOPATH)
../go/src/github.com/librespeed/speedtest/database/bolt/bolt.go:8:2: cannot find package "go.etcd.io/bbolt" in any of:
	/usr/lib/go-1.10/src/go.etcd.io/bbolt (from $GOROOT)
	/home/ubuntu/go/src/go.etcd.io/bbolt (from $GOPATH)
../go/src/github.com/librespeed/speedtest/results/telemetry.go:25:2: cannot find package "golang.org/x/image/font" in any of:
	/usr/lib/go-1.10/src/golang.org/x/image/font (from $GOROOT)
	/home/ubuntu/go/src/golang.org/x/image/font (from $GOPATH)
```
Something's also wrong with package dependency specification?

**Comment by xiruizhao** [comment]
Also, it's helpful to let people know that they can easily use newer versions of go than what is contained in their distribution package index without manually installing tarball archives. 
I am on Ubuntu 18.04.4 LTS, which has go1.10.4. But I can 
```
go get golang.org/dl/go1.14.2
go1.14.2 download # add $HOME/go/bin to $PATH before running this
``` 
and use easily go1.14.2 afterwards.

**Comment by xiruizhao** [comment]
You also have a `.DS_Store` file in `assets`.

**Comment by maddie** [comment]
Thanks for your feedback.

1. As for `go get github.com/librespeed/speedtest`, we're planning to separate the backends into their own repos instead of a branch, that way would make `go get` easier. As for `git checkout`, `-b` works for me but if the correct way of doing it is without `-b` then I'll update the Wiki.

2. Didn't know about `go get golang.org/dl/go1.14.2`, thanks for the tip. I'll add that to the Wiki too.

3. I thought I've ignored `.DS_Store`, perhaps I've added it before the `.gitignore` file. I'll take care of it.

Thanks again.
